libs/config/include/hpx/config.hpp

```diff
@@ -209,7 +209,7 @@
 #  endif
 #endif
 #if !defined(HPX_SPINLOCK_DEADLOCK_DETECTION_LIMIT)
-#  define HPX_SPINLOCK_DEADLOCK_DETECTION_LIMIT 1000000
+#  define HPX_SPINLOCK_DEADLOCK_DETECTION_LIMIT 4294967294
```

Comment:
That's around an hour of 1000 ns sleeps. The old value is a second. Maybe we should have something in between?

Reply:
Yah, this doesn't make sense. @sithhell is this a leftover from testing?

Reply:
the value could be smaller, probably. The old value lead to the detection kicking in though. Note that we don't sleep for 1000 nsecs for every k (only every second iteration when k is large). In any case, this probably needs some adaption.